### ISSUE TITLE
A Link To Updated Sprites On Github

### ISSUE THREAD

**Issue by Bam8647** [Monika-After-Story/MonikaModDev#1759]
Either I'm blind or there doesn't seem to be a link anywhere showing the new sprite codes for creating topics, could you please link them perhaps under **Dialogue Coding** as knowing the right codes to use for expressions is just as important as writing the topics successfully. Thank you!

**Comment by jmwall24** [comment]
There is no cheatsheet yet but it tells you how to use the new codes here: https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/sprite-chart.rpy

At the top is how to do it and every sprite that is currently available is listed from line 1707 to 7399.

**Comment by Bam8647** [comment]
_Only pose number, eyes type, eyebrow type, mouth type are required_

So Does this mean we have to use multiple codes now for each pose?
Where to place her eyebrows, mouth, eyes, and pose?

**Comment by jmwall24** [comment]
Yeah. If you look down toward the bottom at those lines I mentioned that's the format and what each number/letter corresponds to for the poses that have been created.

**Comment by jmwall24** [comment]
The easiest way to see the changes in codes from the old way to the new ways is starting at line 7293.  That shows what all the old poses that we had the cheat sheet for changed to.  For example the default pose of 1 is now 1esa where 1 is steepling pose, e is normal eyes, s is straight eyebrows and a is smiling mouth.

**Comment by ThePotatoGuy** [comment]
There will no longer be a cheatsheet. Contributors should use the [sprite previewer](https://github.com/Monika-After-Story/MonikaModDev/blob/master/FAQ.md#how-do-i-find-the-spritecode-for-an-expression)
